tabbycat/options/views.py

```diff
@@ -23,6 +24,8 @@
 
 
 class TournamentConfigIndexView(AdministratorMixin, TournamentMixin, TemplateView):
+    view_permission = Permission.VIEW_TOURNAMENTPREFERENCEMODEL
+    edit_permission = Permission.EDIT_TOURNAMENTPREFERENCEMODEL
```

Comment:
There is no "editing" on this page, and there is more than just editing preferences on that page. I think we could leave it public, with more precise permissions on the links.

Reply:
I quite don't understand what you meant by "leave it public", because I assume that the configuration is only available in Admin views. Do I just leave the `view_permission` there?

Reply:
Leave it without assigning any permission (edit nor view)